src/modules/logicModule/LM_PC_FundingPot_v1.sol

```diff
@@ -272,53 +355,243 @@ contract LM_PC_FundingPot_v1 is
+
+        AccessCriteriaType accessCriteriaType =
+            AccessCriteriaType(accessCriteriaId_);
+        round.accessCriterias[accessCriteriaId_].accessCriteriaType =
+            accessCriteriaType;
+        round.accessCriterias[accessCriteriaId_].nftContract = nftContract_;
+        round.accessCriterias[accessCriteriaId_].merkleRoot = merkleRoot_;
+
+        for (uint i = 0; i < allowedAddresses_.length; i++) {
+            round.accessCriterias[accessCriteriaId_].allowedAddresses[allowedAddresses_[i]]
+            = true;
+        }
+
+        emit AccessCriteriaEdited(roundId_, accessCriteriaId_);
+    }
+
+    /// @inheritdoc ILM_PC_FundingPot_v1
+    function setAccessCriteriaPrivileges(
+        uint64 roundId_,
+        uint8 accessCriteriaId__,
+        uint personalCap_,
+        uint capByNFT_,
+        uint capByMerkle_,
+        uint capByList_,
+        bool overrideContributionSpan_,
+        uint start_,
+        uint cliff_,
+        uint end_
+    ) external onlyModuleRole(FUNDING_POT_ADMIN_ROLE) {
         Round storage round = rounds[roundId_];
 
+        uint highestCap;
+
         _validateEditRoundParameters(round);
 
-        round.accessCriterias[accessCriteriaId_] = accessCriteria_;
+        if (
+            round.accessCriterias[accessCriteriaId__].accessCriteriaType
+                == AccessCriteriaType.OPEN
+        ) {
+            highestCap = personalCap_;
+        }
+
+        if (
+            round.accessCriterias[accessCriteriaId__].accessCriteriaType
+                == AccessCriteriaType.NFT && capByNFT_ > 0
+        ) {
+            uint nftCap = personalCap_ + capByNFT_;
+            if (nftCap > highestCap) {
+                highestCap = nftCap;
+            }
+        }
+
+        if (
+            round.accessCriterias[accessCriteriaId__].accessCriteriaType
+                == AccessCriteriaType.MERKLE && capByMerkle_ > 0
+        ) {
+            uint merkleCap = personalCap_ + capByMerkle_;
+            if (merkleCap > highestCap) {
+                highestCap = merkleCap;
+            }
+        }
+
+        if (
+            round.accessCriterias[accessCriteriaId__].accessCriteriaType
+                == AccessCriteriaType.LIST && capByList_ > 0
+        ) {
+            uint listCap = personalCap_ + capByList_;
+            if (listCap > highestCap) {
+                highestCap = listCap;
+            }
+        }
+
+        if (!_validTimes(start_, cliff_, end_)) {
+            revert Module__LM_PC_FundingPot__InvalidTimes();
+        }
+
+        AccessCriteriaPrivileges storage accessCriteriaPrivileges =
+            accessCriteriaPrivileges[roundId_][accessCriteriaId__];
+
+        accessCriteriaPrivileges.personalCap = personalCap_;
+        accessCriteriaPrivileges.overrideContributionSpan =
+            overrideContributionSpan_;
+        accessCriteriaPrivileges.start = start_;
+        accessCriteriaPrivileges.cliff = cliff_;
+        accessCriteriaPrivileges.end = end_;
```

Comment:
This is shadowing an existing state var, but the state var should be renamed anyways to conform with our conventions :)